interface/wx/textctrl.h

```diff
@@ -422,6 +433,16 @@ class wxTextAttr
     */
     bool GetFontUnderlined() const;
 
+    /**
+        Returns the underline type, which is one of the @wxTextAttrUnderlineType values
```

Comment:
Wrong tag:
```suggestion
        Returns the underline type, which is one of the @c wxTextAttrUnderlineType values
```
or maybe better `::`. Did you try generating the docs?